regression/libcprover-cpp/call_bmc.cpp

```diff
@@ -6,6 +6,8 @@
 #include <libcprover-cpp/api.h>
 #include <libcprover-cpp/options.h>
 
+#include "goto_model.h"
```

Comment:
We shouldn't be including internal headers in your regression test.

Reply:
Good catch - this is a cleanup leftover from some manual testing we were doing before.

Gone now.